Documentation/TimeSlider/README.md

```diff
@@ -0,0 +1,126 @@
+#TimeSlider
+
+The Time Slider provides controls that allow you to visualize temporal data for time enabled layers in a map or scene.
+
+###Features of the Time Slider
+
+![timeslider](Images/timeslider.png)
+
+* Tap the **Play** button to play a time animation that steps through your data sequentially. Playback interval, direction and whether to repeat or reverse can be configured from the time slider properties. While playing, the **Pause** button displays in its place.
+* Tap the **Forward** button to step forward to the next time stamp.
+* Tap the **Back** button to step back to the previous time stamp.
+* Drag the **Lower Thumb** (Current Extent Start Time) or **Upper Thumb** (Current Extent End Time) to the right or left to step through your temporal data interactively.
+
+#### Slider Thumbs
+
+How the Time Slider's thumbs are presented will vary depending on whether the current time extent represents a time window or instant and whether the start or end time is pinned.
+
+* **Time Window**: If the current time extent represents a non-instantaneous duration of time, and neither the start nor end time is pinned, then two thumbs will be shown on the slider, as shown below. Both thumbs can be manipulated by end users.
+
+  ![time-window](Images/time-window.png)
+
+* **Time Window - Start or End Time Pinned:** If the start or end time is pinned using the `isStartTimePinned` or `isEndTimePinned`, then the corresponding slider thumb is replaced with a bar to visually indicate that the start or end time cannot be manipulated. This allows user to see cumulatively view all data from the pinned thumb until the specified position of other thumb.
```

Comment:
> "This allows user to see cumulatively view all data"

Should remove the word `see`, so the above becomes `This allows user to cumulatively view all data`

Reply:
fixed